src/main/java/com/example/eworldaccessrequest/controller/AccessController.java

```diff
@@ -59,17 +56,19 @@ public EmployeeDTO fetchEmployeeByEmail(@PathVariable("email") String email) {
         return employeeService.fetchEmployeesByAccessGroupID(accessGroupID);
     }
 
-    // Find All Employees with Expired Access Groups
-    @GetMapping("/employee/expired")
-    public List<EmployeeDTO> fetchEmployeesWithExpiredDHSForms() {
-        return employeeService.fetchEmployeesWithExpiredDHSForms();
-    }
-
-    // Find All Employees with Soon-to-Be Expired Access Groups
-    @GetMapping("/employee/soon-expired")
-    public List<EmployeeDTO> fetchEmployeesWithSoonToBeExpiredDHSFormsInOneMonth() {
-        return employeeService.fetchEmployeesWithSoonToBeExpiredDHSFormsInOneMonth();
-    }
+//    // Find All Employees with Expired Access Groups
+//    @Deprecated
+//    @GetMapping("/employee/expired")
+//    public List<EmployeeDTO> fetchEmployeesWithExpiredDHSForms() {
+//        return employeeService.fetchEmployeesWithExpiredDHSForms();
+//    }
+//
+//    // Find All Employees with Soon-to-Be Expired Access Groups
+//    @Deprecated
+//    @GetMapping("/employee/soon-expired")
+//    public List<EmployeeDTO> fetchEmployeesWithSoonToBeExpiredDHSFormsInOneMonth() {
+//        return employeeService.fetchEmployeesWithSoonToBeExpiredDHSFormsInOneMonth();
+//    }
```

Comment:
you can use multi-line comments instead:
```
/*
stuff to comment out
*/
```
This will mean less line changes in the PR

Reply:
thanks, fixed.